index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+
+<body>
+    <main class="top-section">
+        <section class="header">
+            <div class="app-bar">
+                <ul>
+                    <li class="app-bar-item1"><img src="/images/icons/time.png" alt="Time"></li>
+                    <li class="app-bar-item2"><img src="/images/icons/Group4.png" alt="Phone Status"></li>
+                </ul>
+            </div>
+            <nav class="menu-icon">
+                <ul>
+                    <li class="menu-item1"><img src="/images/icons/logo.png" alt="Logo"></li>
+                    <li class="menu-item2"><img src="/images/icons/Union.png" alt="Main menu"></li>
+                </ul>
+            </nav>
+        </section>
+        <section class="headline">
+            <div class="headline-title">
+                Hey There<br />
+                I'm Eric.
+            </div>
+            <div class="headline-title2">I'm A Software Developer!</div>
+            <p class="headline-description">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+            </p>
+            <ul class="headline-social">
+                <li><img src="/images/icons/twitter.png" alt="twitter"></li>
+                <li><img src="/images/icons/linkedin.png" alt="linkedIn"></li>
+                <li><img src="/images/icons/medium.png" alt="medium"></li>
+                <li><img src="/images/icons/git.png" alt="github"></li>
+                <li><img src="/images/icons/angelist.png" alt="angellist"></li>
+            </ul>
```

Comment:
You did a great job wrapping your social icons in a `ul` tag, please each icon should be in a `a href` tag` in the `li` tag , like this 

>     <li> <a></a></li>

Also, in the styling, the icons and the text should have spacing similar to that in the design and it shouldn't cluster with the background image. kindly remember to use the right `font type ` as well 😄 

Figma design :
![image](https://user-images.githubusercontent.com/20537712/235202292-0ab87095-e078-40d4-9776-764770252baf.png)

Your design :
![image](https://user-images.githubusercontent.com/20537712/235202427-06e9516a-9e61-43be-8fc3-3385c4dc8e61.png)